packages/account/src/Sections/Profile/PhoneVerification/phone-verification-page.tsx

```diff
@@ -1,18 +1,26 @@
 import React from 'react';
 import './phone-verification.scss';
 import { LabelPairedArrowLeftCaptionFillIcon } from '@deriv/quill-icons';
-import { Text } from '@deriv-com/quill-ui';
+import { Text, UseTheme } from '@deriv-com/quill-ui';
 import { Localize } from '@deriv/translations';
 import ConfirmPhoneNumber from './confirm-phone-number';
 import ConfirmYourEmail from './confirm-your-email';
 import CancelPhoneVerificationModal from './cancel-phone-verification-modal';
+import { observer, useStore } from '@deriv/stores';
 
-const PhoneVerificationPage = () => {
+const PhoneVerificationPage = observer(() => {
     const [show_email_verification, shouldShowEmailVerification] = React.useState(true);
     const [should_show_cancel_verification_modal, setShouldShowCancelVerificationModal] = React.useState(false);
     const handleBackButton = () => {
         setShouldShowCancelVerificationModal(true);
     };
+    const { ui } = useStore();
+    const { is_dark_mode_on } = ui;
+    const { toggleTheme } = UseTheme();
+
+    React.useEffect(() => {
+        toggleTheme();
+    }, [is_dark_mode_on]);
```

Comment:
Can we add the `<ThemeProvider>` inside `<Routes>` instead so from there we can have an observable `is_dark_mode_on` and we won't be needing `toggleTheme` 

```
<ThemeProvider theme={is_dark_mode_on ? 'dark' : 'light'}>
        <BinaryRoutes is_logged_in={is_logged_in} is_logging_in={is_logging_in} />
    </ThemeProvider>

Reply:
sure let me try it there 👌